webview/src/shared/components/contextMenu/ContextMenu.tsx

```diff
@@ -58,6 +60,8 @@ export const ContextMenu: React.FC<ContextMenuProps> = ({
     placement={'bottom'}
     interactive
     onTrigger={positionContextMenuAndDisableEvents}
+    onClickOutside={(instance: Instance) => instance.hide()}
+    hideOnClick={false}
```

Comment:
[F] Get tippy to not close the tooltip for you if you click inside the row but instead move the context menu if you right click.